plugins/woocommerce/client/blocks/assets/js/blocks/cart-checkout-shared/payment-methods/only-express-payments/index.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import NoticeBanner from '@woocommerce/base-components/notice-banner';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+/**
+ * Render content when no payment methods are found depending on context.
+ */
+const OnlyExpressPayments = () => {
+	return (
+		<NoticeBanner
+			isDismissible={ false }
+			className="wc-block-checkout__only-express-payments-notice"
+			status="info"
+		>
+			{ __(
+				'Only express payment methods are available for this order. Please choose one to place your order.',
```

Comment:
```suggestion
				'Only express payment methods are available for this order. Please select one to continue.',
```
I think the repeated "order" breaks my brain a little. What do you think about this suggestion?